test/unit/registry.test.ts

```diff
@@ -96,7 +96,7 @@ describe('Registry utilities', () => {
       expect(firstRoute!.tokens.length).to.be.greaterThan(0);
       const noRoutes = await registry.getWarpRoutes({ symbol: 'NOTFOUND' });
       expect(Object.keys(noRoutes).length).to.eql(0);
-    });
+    }).timeout(10_000);
```

Comment:
is this seconds? or ms

Reply:
milliseconds, you can check the other configured timeouts in this file. The default is `2_000` and that gets exceeded pretty frequently for this test now